samples/snippets/create_training_pipeline_image_classification_sample.py

```diff
@@ -14,8 +14,8 @@
 
 # [START aiplatform_create_training_pipeline_image_classification_sample]
 from google.cloud import aiplatform
-from google.protobuf import json_format
-from google.protobuf.struct_pb2 import Value
+from google.cloud.aiplatform.v1beta1.schema.trainingjob import definition
+ModelType = definition.AutoMlImageClassificationInputs().ModelType
```

Comment:
I think we don't have to create an instance of `AutoMlImageClassificationInputs` here.

Reply:
Removed.